opentreemap/treemap/templates/instance_base.html

```diff
@@ -158,9 +161,14 @@
            disabled="disabled"><i class="icon-plus"></i> {% trans "Add a" %} {{ term.Resource.singular }}</a>
         {% endif %}
       <a class="btn btn-primary addBtn"
-         data-action='addtree'
+         {% if embed %}
+         target="_blank"
+         {% else %}
          data-feature="add_plot"
```

Comment:
Is it important that we remove these data attributes here if we are in embed mode? The "Add Resource" button only has an if embed/endif around the `target="_blank"`. We may want to keep them the same for consistency.